test/client/scraper/test_drive_metadata.py

```diff
@@ -0,0 +1,19 @@
+import unittest
+from client.api import app
+import json
+
+
+class TestDriveMetadata(unittest.TestCase):
+
+    doc_url = "https://docs.google.com/document/d/1FUo-Him4W6qR96_a6Ftx8skefgAe1vHjDPCfUIxtADM/edit"
```

Comment:
don't use an actual doc_url, instead looking into mocking the response:
https://docs.python.org/3/library/unittest.mock.html